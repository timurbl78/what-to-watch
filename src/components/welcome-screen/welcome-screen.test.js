import React from 'react';
import renderer from 'react-test-renderer';
import WelcomeScreen from './welcome-screen.jsx';

it(`WelcomeScreen correctly renders after relaunch`, () => {
  const tree = renderer
  .create(<WelcomeScreen
    movies={[`testName`]}
  />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
