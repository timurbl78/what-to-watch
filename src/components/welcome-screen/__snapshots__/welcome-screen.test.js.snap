// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeScreen correctly renders after relaunch 1`] = `
<div
  className="page-content"
>
  <section
    className="catalog"
  >
    <h2
      className="catalog__title visually-hidden"
    >
      Catalog
    </h2>
    <ul
      className="catalog__genres-list"
    >
      <li
        className="catalog__genres-item catalog__genres-item--active"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          All genres
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Comedies
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Crime
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Documentary
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Dramas
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Horror
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Kids & Family
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Romance
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Sci-Fi
        </a>
      </li>
      <li
        className="catalog__genres-item"
      >
        <a
          className="catalog__genres-link"
          href="#"
        >
          Thrillers
        </a>
      </li>
    </ul>
    <div
      className="catalog__movies-list"
    >
      <article
        className="small-movie-card catalog__movies-card"
      >
        <div
          className="small-movie-card__image"
        >
          <img
            alt="Fantastic Beasts: The Crimes of Grindelwald"
            height="175"
            src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
            width="280"
          />
        </div>
        <h3
          className="small-movie-card__title"
        >
          <a
            className="small-movie-card__link"
            href="movie-page.html"
          >
            testName
          </a>
        </h3>
      </article>
    </div>
    <div
      className="catalog__more"
    >
      <button
        className="catalog__button"
        type="button"
      >
        Show more
      </button>
    </div>
  </section>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
